js/app.js

```diff
@@ -1,17 +1,16 @@
-import React, { Component } from 'react';
+import React from 'react';
+import * as firebase from 'firebase';
 import AppNavigator from './appNavigator';
 
-class App extends Component {
+firebase.initializeApp({
+  apiKey: 'AIzaSyCY5F5QqKqaVTvwvQQf5EQgu2apNIfG1-0',
```

Comment:
Yeah, I added it now since it doesn't cost us anything if someone starts using it. The key will be discarded once proper security has been taken into account. Was going to ask you about how we should handle keys tomorrow :)